lisa/__init__.py

```diff
@@ -1,3 +1,16 @@
 from lisa.version import __version__
 
+# Put all the code inside one function, to allow easy cleanup of the namespace
+# at the end. We definitely don't want to expose these things to the outside
+# world
+def f():
+    import os
+    if os.getenv('LISA_DO_NOTEBOOK_SETUP'):
```

Comment:
Could we have that done regardless of notebooks? Say I have a script that uses TestEnv.from_cli(), it'd be nice if the logging setup was done for me. The answer probably contains the words "CI", but I'm asking anyway :p

Reply:
No, setting up logging is the job of the application, not of libraries. If for example devlib was messing with logging configuration, that would quickly be an issue. Libraries just use the logging functions, that's not up to them to decide what to do with the result